index.js

```diff
@@ -1,5 +1,11 @@
 /* eslint-disable global-require */
 
+function getDefaultTargets(env) {
+  if (env === 'test') return {node: true, browsers: []}
+  if (env === 'esm' || env === 'cjs') return {node: '12.16', browsers: []}
```

Comment:
Is it worth adding a comment indicating that the `'12.16'` here is just the latest Node LTS version?

Reply:
I'm planning on a diff that annotates the whole preset with comments based on feedback from Edward. Do you mind if I figure out how to phrase this there?